src/features/identityCheck/pages/errors/EduConnectErrors.tsx

```diff
@@ -3,20 +3,13 @@ import { useRoute } from '@react-navigation/native'
 import { withEduConnectErrorBoundary } from 'features/identityCheck/errors/eduConnect/EduConnectErrorBoundary'
 import { EduConnectError } from 'features/identityCheck/errors/eduConnect/types'
 import { EduConnectErrorMessageEnum } from 'features/identityCheck/errors/hooks/useNotEligibleEduConnectErrorData'
+import { logoutUrlIfAllowed } from 'features/identityCheck/utils/logoutUrlIfAllowed'
 import { UseRouteType } from 'features/navigation/RootNavigator'
 
-const allowedDomains = [
-  'https://educonnect.education.gouv.fr',
-  // environnement hors production
-  'https://pr4.educonnect.phm.education.gouv.fr',
-]
-
 export const EduConnectErrors = withEduConnectErrorBoundary(() => {
   const { params } = useRoute<UseRouteType<'EduConnectErrors'>>()
   const logoutUrl = params.logoutUrl
-  if (logoutUrl && allowedDomains.find((domain) => new RegExp(`^${domain}`, 'i').test(logoutUrl))) {
-    globalThis.window.open(logoutUrl)
-  }
+  logoutUrlIfAllowed(logoutUrl)
```

Comment:
```suggestion
  openLogoutUrlIfAllowed(logoutUrl)
```
sinon ça porte un peu à confusion, on sait pas trop ce que ça fait "logoutUrl...()"

Ou alors juste `logoutFromEduConnectIfAllowed` mais un peu plus long, ou `logoutIfAllowed`? Dans tous les cas il faudrait une action dans le naming non?